### ISSUE TITLE
Omni search page

### ISSUE THREAD

**Issue by AlexMountain** [trevorskaggs/shelterly#185]
Searching seems like it's going to be a very common action in shelterly.  Let's up it's accessibility by adding a Search link in the sidebar that takes the user to a page with all searches accessible.  The default search should probably be animals, but then we should have some kind of toggle somewhere on the page to switch between Animal, SR, DA, and Owner searches.  Use the existing search forms and keep the existing search links in each respective app for the time being.

**Comment by AlexMountain** [comment]
We might want the "public" search to go in the sidebar, so let's put this on hold until we determine our final search strategy.
